types/terra-spacer/lib/__spacerShorthand.d.ts

```diff
@@ -0,0 +1,11111 @@
+export type SpacerShorthand =
+  | 'none'
```

Comment:
You could reduce this to about 20 lines of code if you use template literal types from TS 4.1. Unfortunately that would prevent people on older TS versions from using the package, so you'd have to decide if it's worth it.

Reply:
I considered that but 4.1 seemed too recent in terms of version support so decided to opt for this mess 😅. @Berkyle do you have an opinion on this?

Reply:
Probably best to be backwards compatible with 3.x, I don't think we'll be able to enforce 4.x or higher across terra. This is a good note, though